### PR TITLE
Bugfix of offchain worker not properly executed on issue #24

### PR DESCRIPTION
**Pull request by h4n0**
Bugfix of offchain worker not properly executed on issue #24

### CODE REVIEW

**Review by suinuj — APPROVED** [review]

File: node/src/service.rs [hunk 1]
```diff
@@ -147,6 +147,12 @@ where
 	let telemetry_span = TelemetrySpan::new();
 	let _telemetry_span_entered = telemetry_span.enter();
 
+  if parachain_config.offchain_worker.enabled {
```
Comment (geastwood) [inline node/src/service.rs:150]: indentation seems off.

Comment (h4n0) [reply node/src/service.rs:150]: Checked locally, it's tab vs spaces issue. Will write an issue separately about this.

Comment (h4n0) [reply node/src/service.rs:150]: #27  editorconfig support needs to be added

File: ts-tests/tests/utils.ts [hunk 2]
```diff
@@ -188,10 +188,11 @@ export async function launchLitentryNode(
     `--ws-port`,
     `9844`,
     `--alice`,
-    `--`,
     `--execution`,
-    //`wasm`,
     `native`,
+    `--`,
```
Comment (Satoshi-Kusumoto) [inline ts-tests/tests/utils.ts:193]: what's the difference with the original one?

**Pull request merged**